tofhir-server/src/main/scala/io/tofhir/server/model/SchemaDefinition.scala

```diff
@@ -18,12 +20,21 @@ case class SchemaDefinition(id: String = UUID.randomUUID().toString,
                             name: String,
                             rootDefinition: Option[SimpleStructureDefinition],
                             fieldDefinitions: Option[Seq[SimpleStructureDefinition]]) {
+
+
   /**
-   * Copies this schema with only the metadata attributes
+   * Retrieves metadata for this schema. The metadata is being used in the folder-based implementation for the time being.
    *
    * @return
    */
-  def copyAsMetadata(): SchemaDefinition = {
-    copy(this.id, this.url, this.`type`, this.name, None, None)
+  def getMetadata(): JObject = {
```

Comment:
Since **getMetadata** methods are only used in **ProjectFolderRepository** and they are specific to folder-based implementation, I think we can remove them and write a method/methods in **ProjectFolderRepository** to extract metatada information.  WDYT ?